<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modPrzejazd</class>
 <widget class="QDialog" name="modPrzejazd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>410</y>
     <width>194</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="Ok">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="anuluj">
      <property name="text">
       <string>Anuluj</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>361</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="numerParagonuLabel">
      <property name="text">
       <string>Numer paragonu</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="numerParagonuLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="kierowcaLabel">
      <property name="text">
       <string>Kierowca</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="kierowcaBox"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="numerTelefonuKlientaLabel">
      <property name="text">
       <string>Numer telefonu klienta</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="numerTelefonucomboBox"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="stawkaLabel">
      <property name="text">
       <string>Stawka</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="stawkaComboBox"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="dataRozpoczeciaLabel">
      <property name="text">
       <string>Data rozpoczecia</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QDateTimeEdit" name="dataRozpoczeciaDateTimeEdit">
      <property name="displayFormat">
       <string>yyyy-MM-dd HH:mm:ss</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="dataRozpoczeciaLabel_2">
      <property name="text">
       <string>Data zakończenia</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QDateTimeEdit" name="dataZakonczeniaDateTimeEdit">
      <property name="displayFormat">
       <string>yyyy-MM-dd HH:mm:ss</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="punktPoczTkowyLabel">
      <property name="text">
       <string>Punkt początkowy</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="punktPoczTkowyLineEdit"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Punkt końcowy</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="punktKoncowyLineEdit"/>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="dUgoTrasyLabel">
      <property name="text">
       <string>Długość trasy</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QDoubleSpinBox" name="dUgoTrasyDoubleSpinBox">
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="maximum">
       <double>10000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="cenaLabel">
      <property name="text">
       <string>Cena</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QDoubleSpinBox" name="cenaDoubleSpinBox">
      <property name="maximum">
       <double>99999.990000000005239</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>361</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QComboBox" name="nrPCBwyb"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Wybierz numer paragonu przejazdu do modyfikacji:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
